### PR TITLE
Make AEX boot animation work

### PR DESCRIPTION
**Pull request by KNnut**

### CODE REVIEW

File: aosp_wt88047.mk [hunk 1]
```diff
@@ -36,5 +36,4 @@ PRODUCT_BUILD_PROP_OVERRIDES += \
 BUILD_FINGERPRINT := Xiaomi/chiron/chiron:8.0.0/OPR1.170623.027/V9.5.4.0.ODEMIFA:user/release-keys
 
 # Boot animation
-TARGET_SCREEN_HEIGHT := 1280
-TARGET_SCREEN_WIDTH := 720
\ No newline at end of file
+TARGET_BOOT_ANIMATION_RES := 720
```
Comment (karthikkm007) [inline aosp_wt88047.mk:39]: cool

File: device.mk [hunk 2]
```diff
@@ -280,11 +276,6 @@ PRODUCT_COPY_FILES += \
 PRODUCT_COPY_FILES += \
     frameworks/native/data/etc/android.hardware.audio.low_latency.xml:$(TARGET_COPY_OUT_VENDOR)/etc/permissions/android.hardware.audio.low_latency.xml
 
-# Boot animation
-TARGET_SCREEN_HEIGHT := 1280
-TARGET_SCREEN_WIDTH := 720
-TARGET_BOOTANIMATION_HALF_RES := true
```
Comment (karthikkm007) [inline device.mk:286]: isn't it worked?

Comment (KNnut) [reply device.mk:286]: [they](https://github.com/LineageOS/android_vendor_lineage/blob/lineage-15.1/bootanimation/Android.mk) only work in lineage
`TARGET_BOOT_ANIMATION_RES` is for [AEX](https://github.com/AospExtended/platform_vendor_aosp/blob/8.1.x/config/bootanimation.mk)

Comment (adithya2306) [reply device.mk:286]: > [they](https://github.com/LineageOS/android_vendor_lineage/blob/lineage-15.1/bootanimation/Android.mk) only work in lineage
> `TARGET_BOOT_ANIMATION_RES` is for [AEX](https://github.com/AospExtended/platform_vendor_aosp/blob/8.1.x/config/bootanimation.mk)

Yup thats true

Comment (karthikkm007) [reply device.mk:286]: okey then
merged

**Pull request merged**